core/electron/src/common/ElectronIpcInterface.ts

```diff
@@ -15,5 +15,6 @@ export const electronIpcStrings = {
 
 /** Asynchronous methods of dialog module in an Electron app.
  * @beta
+ * @deprecated in 5.0. Use `Electron.Dialog` from `electron` instead.
```

Comment:
Since the API is beta, should we just remove it?

Reply:
I don't think anyone is using this, so no one would notice it's removal. 

That being said, I would like to follow our deprecated -> remove process as much as possible. As this is just a TS type, there is no maintenance required here.

Reply:
Ok thats fine, but just keep in mind the deprecation policy is targeted mainly towards public apis